Parsers/ASimNetworkSession/ARM/ASimNetworkNotables/README.md

```diff
@@ -0,0 +1,15 @@
+# Source agnostic ASIM Network Notables Normalization Parser
+
+This template deploys the ASIM NetworkSession schema parser for Source agnostic. The parser is a part of the Advanced SIEM Information Model.
```

Comment:
Just a reminder that this my example for the need to update the generation script for readme files.

Reply:
I did not generate readme files now. Duplicated the existing one. But I agree, we need to update the readme template  for the generator.